src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -1,3 +1,8 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ */
```

Comment:
@dotnet/aspnet-build any reason why we may be excluding the license from .ts files?

Reply:
Umm, because the analyzer doesn't know about TS files (yet)

Reply:
https://github.com/dotnet/aspnetcore/pull/39164